rest_sensors.cpp

```diff
@@ -1687,10 +1705,9 @@ int DeRestPluginPrivate::changeSensorConfig(const ApiRequest &req, ApiResponse &
                             updated = true;
                         }
                     }
-                    else if (devManaged && rsub)
+                    else if (devManaged && rsub) // Managed by DDF ?
                     {
-                        data.uinteger = data.boolean; // Use integer representation
-                        change.addTargetValue(rid.suffix, data.uinteger);
+                        change.addTargetValue(rid.suffix, data.boolean);
                         rsub->addStateChange(change);
                         updated = true;
```

Comment:
Please leave the code as it was, there's no necessity to change it. Changing it may cause unforseen issues.

Reply:
Ok, have make rollback.